x-pack/platform/plugins/shared/streams_app/public/components/esql_chart/controlled_esql_chart.tsx

```diff
@@ -65,7 +65,7 @@ export function ControlledEsqlChart<T extends string>({
   xDomain: customXDomain,
 }: {
   id: string;
-  result: AbortableAsyncState<UnparsedEsqlResponse>;
+  result: AbortableAsyncState<ESQLSearchResponse>;
```

Comment:
This return format might be different, not sure, I assume you validated it though. I personally struggle quite a bit with the default response type because it's so loosely typed.